src/domain/app/useStaticLinks.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+
+import useGetPathname from '../../common/route/utils/useGetPathname';
+
+// todo: this hook is used for static links in the footer navigation
+// links should be reusable so the header language switch knows how to switch the language accordinly
+// if possible, replace with header menu items from headless cms
```

Comment:
```suggestion
// links should be reusable so the header language switch knows how to switch the language accordingly
// if possible, replace with footer menu items from headless cms
```
typo: accordinly -> accordingly
"replace with header menu items" -> "replace with footer menu items", right, or was there a reason for using `header` here that I'm overlooking?